programmers/문자열-내-p와-y의-갯수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var answer = true;
+    var pcount = 0;
+    var ycount = 0;
+    for (var i in s){
+        if (s[i] == 'p' || s[i] == 'P'){
+            pcount++;
+        }
+        else if (s[i] == 'y' || s[i] == 'Y'){
+            ycount++;
+        }
+    }
+    if (pcount != ycount) {
```

Comment:
자바스크립트에서는 ==연산자와 ===연산자가 있습니다. ==연산자는 동등 연산자이고 ===연산자는 일치연산자 인데 ===가 좀 더 정확하게 비교를 해주기때문에 특별한 상황 아니면 ===연산자를 이용해주세요.

==, ===연산자 차이: http://guswnsxodlf.github.io/javascript-equal-operator
```suggestion
    if (pcount !== ycount) {
```